wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
+1. **Glissez et déposez les deux icônes dans le dossier Applications** pour installer Jamulus.
+1. **Executez Jamulus**. Vous devriez maintenant pouvoir utiliser Jamulus comme n'importe quelle autre application. Si vous voyez un message qui vous dit que Jamulus ne peut pas être ouvert parce que le développeur ne peut pas être vérifié, lisez la suite et jetez un coup d'oeil à la [section suivante](#jamulus-ne-peut-pas-être-ouvert-car-le-développeur-ne-peut-pas-être-vérifié).
+
+_Vous pouvez supprimer le dossier dans le répertoire Downloads contenant le `.dmg` et éjecter le lecteur "Jamulus" de votre bureau. Ils ne sont plus nécessaires._
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous en avez envie.
+
+## "Jamulus" ne peut pas être ouvert car le développeur ne peut pas être vérifié
+
+**La première fois que vous essaierez de double-cliquer sur Jamulus pour l'exécuter, vous verrez probablement un message disant qu'il ne peut pas être ouvert.**
+
```

Comment:
* same as above: "ouvert -> exécuté"
* it's missing a whole part of the sentence: "since we don't code-sign Jamulus yet (it's quite expensive) -> "puisque nous n'avons pas encore signé le code de Jamulus (c'est assez cher)"

Reply:
No, the message in the alert is « Jamulus ne peut pas être ouvert car …»

So it should be:
a première fois que vous essaierez de double-cliquer sur Jamulus pour l'ouvrir, vous verrez probablement un message disant qu'il ne peut pas être ouvert.

On MacOS we "open" an Application. MacOS is a strange world sometimes. ;)

Reply:
>     * it's missing a whole part of the sentence: "since we don't code-sign Jamulus yet (it's quite expensive) -> "puisque nous n'avons pas encore signé le code de Jamulus (c'est assez cher)"

It is not in the `translation`branch. Only in `changes` for the moment. If Y've good understand the reference for translation is the `translation`branch. So we must wait for the changes to add it. No?

Reply:
Yes. Please wait for the translation (at least that's the current policy).

Reply:
@ann0see 
> Yes. Please wait for the translation (at least that's the current policy).

Thank you for clarifying this again.
So I resolve the conversation.

Reply:
> No, the message in the alert is « Jamulus ne peut pas être ouvert car …»

OK, strange to my eyes, but if it matches what the user is actually seeing, no worries at all.

> It is not in the translationbranch.

My mistake then.